src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateUserSpecificRegistrySettings(CommandSettings command, string
 
         private void UpdateScreenSaverSettings(CommandSettings command, string securityId)
         {
-            if(!command.GetDisableScreenSaver())
+            if(command.GetKeepScreenSaver())
```

Comment:
no. asking user's consent for this is the right thing as with screen saver disabled the machine will not get locked and it should be user's choice then. And if we don't disable screen saver it doesn't mean the UI tests will fail always.